sql/moz-fx-data-shared-prod/telemetry_derived/desktop_engagement_clients_v1/query.sql

```diff
@@ -12,20 +12,22 @@ SELECT
   cfs.attribution_dlsource,
   cfs.attribution_medium,
   cfs.attribution_ua,
+  cfs.attribution_experiment,
   cls.normalized_channel,
   mozfun.norm.os(cls.os) AS normalized_os,
   COALESCE(
     mozfun.norm.windows_version_info(cls.os, cls.os_version, cls.windows_build_number),
     NULLIF(SPLIT(cls.normalized_os_version, ".")[SAFE_OFFSET(0)], "")
   ) AS normalized_os_version,
+  cls.startup_profile_selection_reason_first,
```

Comment:
I'm thinking about renaming this to startup_profile_selection_reason, which is the name in some other tables.
My understanding is that in theory this shouldn't change.

Reply:
I'm for it, it's nicer shorter anyway